src/components/shared/Svg.css

```diff
@@ -353,6 +360,30 @@ html[dir="rtl"] .arrow,
   visibility: hidden;
 }
 
+.webpack svg {
```

Comment:
Were these changes intended?

Reply:
It looks like I made a mistake with the mergetool. I've (hopefully) fixed that and I cleaned up the commits. Let me know if there are any other problems and I'll get them fixed!